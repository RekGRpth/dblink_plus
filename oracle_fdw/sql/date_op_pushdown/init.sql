\! sqlplus -s system/manager@orcl @sql/date_op_pushdown/init_ora.sql
