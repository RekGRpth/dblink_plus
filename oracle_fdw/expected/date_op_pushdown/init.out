\! sqlplus -s system/manager@orcl @sql/date_op_pushdown/init_ora.sql

Session altered.


Table truncated.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


Table truncated.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


Table truncated.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


Table truncated.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


Table truncated.


1 row created.


Table truncated.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


Table truncated.


1 row created.


Table truncated.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


Table truncated.


1 row created.


Table truncated.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


Table truncated.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


Table truncated.


1 row created.


Session altered.


Session altered.


Session altered.


Session altered.


Table truncated.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.

